homeassistant/components/bluesound/strings.json

```diff
@@ -1,4 +1,25 @@
 {
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]"
```

Comment:
Can we use data_description to give more context? Please check other integrations for examples